TradeItIosTicketSDK2/TradeItLinkedBrokerManager.swift

```diff
@@ -325,6 +325,8 @@ import PromiseKit
 
     private func loadLinkedBrokerFromLinkedLogin(_ linkedLogin: TradeItLinkedLogin) -> TradeItLinkedBroker {
         let tradeItSession = sessionProvider.provide(connector: self.connector)
+        //provides a default token, so if the user doesn't authenticate before an other call, it will pass an expired token in order to get the session expired error
+        tradeItSession?.token = "dd61aa94fa094e6ab54fa4b31853bbd4"
```

Comment:
In situations like this it's really helpful to use an actual dummy string to help with debugging rather than something that looks like it could be a valid token.

So maybe `trade-it-fetch-fresh-token`